skfda/exploratory/visualization/fpca.py

```diff
@@ -0,0 +1,67 @@
+from matplotlib import pyplot as plt
+from skfda.representation import FDataGrid, FDataBasis, FData
+
+
+def plot_fpca_perturbation_graphs(mean, components, multiple,
+                                  fig: plt.figure = None, **kwargs):
+    """ Plots the perturbation graphs for the principal components.
+    The perturbations are defined as variations over the mean. Adding a multiple
+    of the principal component curve to the mean function results in the
+    positive perturbation and subtracting a multiple of the principal component
+    curve results in the negative perturbation. For each principal component
+    curve passed, a subplot with the mean and the perturbations is shown.
+
+    Args:
+        mean (FDataGrid or FDataBasis):
+            the functional data object containing the mean function
+        components (FDataGrid or FDataBasis):
+            the principal components
+        multiple (float):
+            multiple of the principal component curve to be added or
+            subtracted.
+        fig (figure object, optional):
+            figure over which the graph is plotted. If not specified it will
+            be initialized
+
+    Returns:
+        (FDataGrid or FDataBasis): this contains the mean function followed
+        by the positive perturbation and the negative perturbation.
+    """
+    if fig is None:
+        fig = plt.figure(figsize=(6, 4 * len(components)))
+    axes = fig.subplots(nrows=len(components))
+
+    for i in range(len(axes)):
+        aux = _get_component_perturbations(mean, components, i, multiple)
```

Comment:
If `len(mean) > 1` I would compute the mean. It is easier for the user.

Reply:
Agreed and done